modules/ROOT/pages/backup-restore/restore-dump.adoc

```diff
@@ -141,14 +141,14 @@ You can load a dump from a local directory using the following command:
 
 [source,shell, role="nocopy"]
 ----
-bin/neo4j-admin database load --from-path=/full-path/data/dumps <database> --overwrite-destination=true
+bin/neo4j-admin database load --from-path=/full-path/data/dumps neo4j --overwrite-destination=true
```

Comment:
or it's better to replace `neo4j` with `mydatabase`? To be consistent with other code examples where `mydatabase` means the DB name (if I understand it correctly)

Reply:
I changed it here to `neo4j` because the command was using `--overwrite-destination=true`.

Reply:
Also, because in the Community edition, we have only the `neo4j` database.